packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -7,9 +7,17 @@ public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(EndpointConfiguration endpointConfiguration)
         {
-            // NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence. 
+            //TODO: NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence.
             // Refer to the documentation for more details on specific options.
             endpointConfiguration.UsePersistence<PLEASE_SELECT_ONE>();
+
+            //TODO: NServiceBus will move messages that fail repeatedly to a separate "error" queue. We recommend 
+            // that you start with a shared error queue for all your endpoints for easy integration with ServiceControl.
+            endpointConfiguration.SendFailedMessagesTo("error");
+
+            //TODO: NServiceBus will store a copy of each successfully process message in a separate "audit" queue. We recommend
```

Comment:
The default settings there are only "wrong" for a transport like MSMQ. From the perspective of a transport like Rabbit, the setting listed there is actually correct to get a shared queue.

Reading it from that perspective, the "we recommend" sentence sounds like we're telling them why we've chosen this value, not that there is something for them to TODO.